### DISCUSSION TITLE
[Feature Request] Copy pathing areas from one worker to another

### DISCUSSION THREAD

**Discussion by tunbridgep** [OpenRCT2/OpenRCT2#15761] [Feature requests]
I think it would be a good idea if we could copy the selected pathing regions from one worker to another (even across disciplines). If I have my park seperated into 2 distinct sections, and I want separate mechanics/handymen to work at different parts of the park, it would be super awesome being able to copy someone else's set patrol area instead of having to redefine it every time. This would be especially useful in the Dinky Park mission, since you literally have 2 seperate parks, and it can take a very long time for a mechanic (for example) to walk from one end of the highway to the other.

The ability to "duplicate" workers would be nice too. That is, hire a new worker of the same type, with the same patrol area, and the same set of duties, with the click of a button.

**Comment by AaronVanGeffen** [comment]
Please *do* file them as different issues. :) Easier to keep track of the inevitable discussion, and of who's working on what.

**Comment by tunbridgep** [comment]
Very well, I will move all but the first to new threads. This is now the "copy pathing" feature request thread :)

**Comment by J-PIE-314** [comment]
This sounds like it would be best implemented as a plug-in.
